beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -96,33 +68,6 @@ pub static SIGNED_BEACON_BLOCK_BELLATRIX_MAX: LazyLock<usize> =
     + types::ExecutionPayload::<MainnetEthSpec>::max_execution_payload_bellatrix_size() // adding max size of execution payload (~16gb)
     + ssz::BYTES_PER_LENGTH_OFFSET); // Adding the additional ssz offset for the `ExecutionPayload` field
 
-pub static SIGNED_BEACON_BLOCK_CAPELLA_MAX: LazyLock<usize> = LazyLock::new(|| {
-    *SIGNED_BEACON_BLOCK_CAPELLA_MAX_WITHOUT_PAYLOAD
-    + types::ExecutionPayload::<MainnetEthSpec>::max_execution_payload_capella_size() // adding max size of execution payload (~16gb)
```

Comment:
Looks like we can also remove the `max_execution_payload_capella_size` functions:

https://github.com/sigp/lighthouse/blob/ecdf2d891fb78d84e500ca8cf3a9deb9706263cf/consensus/types/src/execution_payload.rs#L131-L182

Reply:
https://github.com/sigp/lighthouse/pull/6798/commits/4261ce0e49866782d0297667c41faf6c24faa9cc